checkpoints/checkpoint1/todaysvision/website/css/style.css

```diff
@@ -0,0 +1,213 @@
+header{
+  visibility: hidden;
+}
+
+select{
+  background-color:#A28A69;
+}
+
+body{
+  display: flex;
+  background-color: black;
+  width: 100%;
+  height: 900px;
+  flex-direction: column;
+}
+
+
+main{
+
+display: flex;
+height:700px;
+flex-direction: column;
+background-color: pink;
+}
+
+.header{
+  display: flex;
+  background-color: blue;
+  width: 100%;
+  height:100%;
+  padding: 30px;
+  justify-content: center;
+  flex-direction: column;
+  align-items: center;
+  background-image: url("../images/Bridge1.jpg");
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: center;
+}
+
+.headerbuttons{
+  display: flex;
+  width: 40%;
+  height:30px;
+  justify-content: space-between;
+
+
+}
+.button{
+  background-color:beige;
+  font-family: fantasy;
+  font-size: 15px;
+}
+
+.missionstatement{
+  display: flex;
+  color:beige;
+  font-style:italic;
+  font-family: fantasy;
+  font-size: 50px;
+  align-items:center;
+  justify-content:center;
+  width: 45%;
+  height: auto;
+  margin-top: 100px;
+  font-variant: bold;
+  text-decoration: underline;
+  font-weight: bold;
+
+
+}
+.Pamela{
+  display: flex;
+  background-color:black;
+  width:100%;
+  height:500px;
+  align-items: center;
+  font-family: fantasy;
+  padding-top: 75px;
+  padding-bottom: 70px;
+
+
+}
+aside{
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  background-color:lightgrey;
+  height: 300px;
+  width: 400px;
+
+}
+.Pamelapicture{
+  background-color: black;
+  color: white;
+  width: 250px;
+  height: 450px;
+  background-image: url("../images/Pamela.jpg");
+  background-repeat:no-repeat;
+  background-size: cover;
+  background-position: center;
+}
+
+.words{
+  display: flex;
+  background-color:lightgrey;
+  align-self:center;
+  flex-direction: column;
+  margin-right: 100px;
+  margin-left: 100px;
+  height: 250px;
+  width: 1200px;
+  padding: 5px;
+
+}
+
+footer{
+  display:flex;
+  background-color:lightgrey;
+  flex-direction: column;
+  width: 100%;
+  height: 150px;
+  justify-content:center;
+  align-items:center;
+}
+
+.socialmedia
+{
+  display: flex;
+  justify-content: space-around;
+  width: 100%;
+
+}
+.fa:hover{
+  opacity: 0.7;
+}
+
+.fa{
+  width: 50px;
+
+}
+
+.copyrights{
+  display: flex;
+  background-color:white;
+  width:90%;
+  height:70px;
+  justify-content: center;
+  color:grey;
+
+}
+
+@media (max-width:576px){
+  header{
+  visibility: visible;
+  display: flex;
+  align-items: flex-end;
+  height:30px;
+  width:100%;
+  background-color:lightgrey;
+}
+
+}
+
+@media(max-width:576px){
```

Comment:
The media query part and making the site responsive does not present a clean UI and the transition seems a little awkward.